app/views/roles/_announcements.html.erb

```diff
@@ -0,0 +1,23 @@
+<% if role.announcements %>
```

Comment:
This view and the one for people are very similar, could we merge the two somehow? They'd only need to take a `announcements` object and then could just `announcements.items` and `announcements.links`. I don't think there's anything specific to the person or the role.